sound/soc/sof/sof-priv.h

```diff
@@ -473,7 +473,6 @@ int snd_sof_ipc_get_comp_data(struct snd_sof_ipc *ipc,
 int snd_sof_init_topology(struct snd_sof_dev *sdev,
 			  struct snd_soc_tplg_ops *ops);
 int snd_sof_load_topology(struct snd_sof_dev *sdev, const char *file);
-void snd_sof_free_topology(struct snd_sof_dev *sdev);
 int snd_sof_complete_pipeline(struct snd_sof_dev *sdev,
```

Comment:
maybe add a comment so that there is an explicit note that free_topology is handled by the framework.